src/holodeck/environments.py

```diff
@@ -398,6 +401,25 @@ def set_ticks_per_capture(self, agent_name, ticks_per_capture):
             command_to_send = RGBCameraRateCommand(agent_name, ticks_per_capture)
             self._enqueue_command(command_to_send)
 
+    def rotate_sensor(self, agent_name, sensor_name, rotation):
```

Comment:
Eventually we'll want this method off of the `HolodeckEnvironment` class and probably attached to HolodeckAgent.

For now this is OK